File: app/src/main/java/com/example/denisvodolazskiy/testbottomscreen/BottomActivity.java
package com.example.denisvodolazskiy.testbottomscreen;

import android.os.Bundle;
import android.view.MotionEvent;
import android.view.View;
import android.view.ViewGroup;

import androidx.appcompat.app.AppCompatActivity;

public class BottomActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.content_bottom);

        findViewById(R.id.btnDone).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                BottomActivity.this.finish();
            }
        });

        findViewById(R.id.rlBackGround).setOnTouchListener(new View.OnTouchListener() {
            @Override
            public boolean onTouch(View v, MotionEvent event) {
                BottomActivity.this.finish();
                return false;
            }
        });

        getWindow().setLayout(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT);
    }
}

File: app/src/main/java/com/example/denisvodolazskiy/testbottomscreen/BottomActivity.kt
package com.example.denisvodolazskiy.testbottomscreen

import android.os.Bundle
import android.view.Gravity
import android.view.View
import android.view.ViewGroup
import androidx.appcompat.app.AppCompatActivity

class BottomActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        overridePendingTransition(R.anim.bottom_dialog_anim_in, R.anim.bottom_dialog_anim_out)

        setContentView(R.layout.content_bottom)
        //optional
        setFinishOnTouchOutside(true)

        findViewById<View>(R.id.btnDone).setOnClickListener { finish() }

        window.setLayout(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
        val lp = window.attributes
        lp.gravity = Gravity.BOTTOM
        window.attributes = lp
    }

    override fun onPause() {
        super.onPause()
        if (isFinishing) {
            overridePendingTransition(R.anim.bottom_dialog_anim_in, R.anim.bottom_dialog_anim_out)
        }
    }
}